lib/util/fs/util.ts

```diff
@@ -0,0 +1,13 @@
+import upath from 'upath';
+import { FILE_ACCESS_ERROR } from '../../constants/error-messages';
+import { logger } from '../../logger';
+
+export function assertBaseDir(path: string, baseDir: string): void {
+  if (!path.startsWith(upath.resolve(baseDir))) {
```

Comment:
`baseDir` may also be `.` or `./` (e.g. see [here](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/npm/post-update/yarn.spec.ts#L47)). Without `upath.resolve` the `startsWith` check would fail.

Reply:
We should move the resolve to global config in a later PR